hail/python/hail/expr/aggregators/aggregators.py

```diff
@@ -292,6 +292,26 @@ def approx_quantiles(expr, qs, k=100) -> Expression:
     else:
         return _quantile_from_cdf(approx_cdf(expr, k), qs)
 
+@typecheck(expr=expr_numeric, k=int)
+def approx_median(expr, k=100) -> Expression:
+    """Compute the approximate median.
+
+    ..include: _templates/experimental.rst
+
+    Examples
+    --------
+    Estimate the median of the `HT` field.
+
+    >>> table1.aggregate(hl.agg.approx_median(table1.HT)) # doctest: +SKIP_OUTPUT_CHECK
+    64
```

Comment:
In cases like this, it seems valuable to me to actually document the implementation, to help people discover and learn the more general operations.

Reply:
Good call :+1: I'll mention this is just a wrapper on `approx_quantiles`. Also, forgot to copy the documentation of what `k` does.